src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -532,3 +532,10 @@ def copy_from_household_member(
             )
 
     return pop
+
+
+def write_metadata_file(final_output_dir: Path, label_version: str) -> None:
```

Comment:
Why is this a parquet file? Can't it just be a yaml or json file?

Reply:
I was trying to align with our other outputs from post-processing that are either hdf or parquet. We don't know all of what will be going into this file and what the format should be so I was just trying to stay consistent.